config_example.sh

```diff
@@ -13,40 +15,49 @@ IPV6_KUBECONFIG=ipv6-kubeconfig
 # images referenced by the payload - are hosted
 IPV6_PULLSECRET=ipv6-pullsecret
 
-# machine-config-operator build config
-MCO_STREAM=machine-config-operator
-MCO_GIT_URI=https://github.com/russellb/machine-config-operator.git
-MCO_GIT_REF=ipv6
-MCO_DOCKERFILE=Dockerfile
+## Specific modules
+
+# cluster-dns-operator build config
```

Comment:
more changes in this file than just adding the lines for the new operator, intentional?

Reply:
nevermind, i see you have a commit to alphabetize the list